playwright.config.ts

```diff
@@ -10,9 +10,10 @@ export default defineConfig({
   forbidOnly: !!process.env.CI,
   // TODO hardcoded values should be extracted to ENVs
   retries: process.env.CI ? 2 : 0,
-  workers: process.env.CI ? 2 : undefined,
+  workers: process.env.CI ? 2 : 2,
```

Comment:
if workers don't matter is CI or is 2 would be better to set it like this:
```suggestion
  workers: 2,
```